packages/core/admin/admin/src/pages/App/index.js

```diff
@@ -82,7 +79,7 @@ function App() {
           data: {
             data: { hasAdmin, uuid, menuLogo, authLogo },
           },
-        } = await axios.get(`${strapi.backendURL}/admin/init`);
+        } = await get(`${strapi.backendURL}/admin/init`);
```

Comment:
Could you add a TODO to see why this needs an absolute URL and the others do not?

Reply:
no good catch.